packages/resource/src/collection.ts

```diff
@@ -38,3 +36,15 @@ export const resources = <T, Rel extends string>(
   res
     .map(r => (r.payload.length > 0 ? results<T, Rel>(r) : empty<T, Rel>()))
     .getOrElseValue(none<T, Rel>());
+
+export const hasUrl = <T, Rel extends string>(url: string, rel: Rel) => (
```

Comment:
I'm not a fan of putting this in here because I prefer to discourage the use of booleans. This function should likely be unused once everything in EP is moved to canon and I don't want this to be used anywhere else.

Reply:
Noted, thanks 😄